### ISSUE TITLE
Any C/C++ guides?

### ISSUE THREAD

**Issue by gamethrower** [UnamSanctam/SilentXMRMiner#186]
Once again I'm here. Wanted to ask how did you learn C/C++. Is there any advanced guides on it?
Every guide that I find is either basic shit like math operations or articles that too hard for me to understand.
I'm not a complete fool at programming. I know the basics and was writing some code on Python and Pascal. Any advice?

**Comment by UnamSanctam** [comment]
Well the first project I started to learn C++ with was XMRig since from personal experience I learn best from going through code of things I'm interested in. So I just opened it in Visual Studio/Notepad++ and went through all the code and saw how it all worked together, then I started doing modifications to it and learned over a few days. I never really use any tutorials since I find them too boring or unproductive to remember anything from.

This is probably only possible though since I know a lot of other programming languages and I could understand most things through my understanding of them. My best tip is to find the method of learning that works best for you, some learn best from reverse engineering, some from listening, some from reading and some from just creating things and learning along the way and finding the best method for yourself I personally feel like brings the most gain.
